src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-filter/TranscriptsFilter.scss

```diff
@@ -2,44 +2,35 @@
 @import 'src/content/app/entity-viewer/gene-view/styles/constants';
 
 .container {
-  display: grid;
-  grid-template-columns: $left_column $middle_column_left_gap auto;
-  padding: 9px 0;
-}
-
-.filterLabel {
-  grid-column: 1/2;
-  text-align: right;
-  padding-right: 2px;
-  cursor: pointer;
-}
-
-.chevron {
-  margin-left: 10px;
-  height: 12px;
-  width: 12px;
-  margin-bottom: -3px;
+  grid-area: 3/1/3/4;  
+  position: sticky;
+  height: 100%;
+  top: -5px;
+  z-index: 3;  
 }
 
 .filterBox {
   display: grid;
-  grid-column: 3;
-  grid-template-columns: 270px minmax(210px, auto) 20px;
+  grid-column: 4;
+  grid-template-columns: 160px 270px minmax(210px, auto) 20px;
   background-color: $soft-black;
   color: $white;
   text-align: left;
   padding: 15px 15px 45px 20px;
-  margin-top: 2px;
-  margin-left: 2px;
   font-weight: 300;
+  border-bottom-left-radius: 3px;
+  border-bottom-right-radius: 3px;
+  box-shadow: -1px 3px 9px 2px rgba(0, 0, 0, 0.5);
```

Comment:
Shouldn't have the box-shadow in the top:

![image](https://user-images.githubusercontent.com/6834224/120822073-f2b47b00-c54d-11eb-99ec-52a6deecb583.png)

Reply:
i tried removing it but couldn't. We dont have an option to remove the box-shadow top, you have to play with the shadow positioning, making the y 0px but then there was a bigger shadow which wasnt what Andrea wanted. I had a chat with Andrea about that and she said its fine if we cant remove it. I am happy to remove it if there is a solution.

Reply:
There is.

- Add `overflow: hidden` to the `.container` class of transcripts filter ([link](https://github.com/Ensembl/ensembl-client/pull/501/files/bc9eee191dae073359701aabffb4b31565fbde19#diff-dcb53804ae1693168649be7ed14d41c1080abe4d328c634a35adf2c716d944deR4)). Then add a padding to it (0 on the top, possibly 0 on the left, 9 pixels to the bottom and to the right)
- remove the `background-color` from the `geneViewTabs` class ([link](https://github.com/Ensembl/ensembl-client/pull/501/files/bc9eee191dae073359701aabffb4b31565fbde19#diff-f48b6c74fd08794741e899da71186f70aeb1795c0ef9c9732b2458ec5aba3f92R38)) and move it over to the `tabWrapper` and `filterLabelContainer` class so as to make the rest of the `geneViewTabs` box transparent

Result:

https://user-images.githubusercontent.com/6834224/120832417-8be88f00-c558-11eb-8fcc-522ba517ad1c.mp4

Reply:
I will give it a go, thanks

Reply:
Done

Reply:
Only something needs to be done about the `filterLabelContainer`, because it does not sufficiently cover the area, and labels that get scrolled past it are visible on the other side: 

![seethrough](https://user-images.githubusercontent.com/6834224/121032928-a5c9e200-c7a3-11eb-8227-fe4da2d94779.gif)

Reply:
Good spot, looking into it now.

Reply:
fixed